src/styles/card.css

```diff
@@ -0,0 +1,110 @@
+/* #region card */
+:root {
+  font-family: Roboto, sans-serif;
+
+  --secondary-text-color: #616070;
+  --button-color: #00acdc;
+  --main-border-radius: 5px;
+  --text-color: #060b35;
+}
+
+body,
+h2,
+p,
+div {
+  margin: 0;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 198px;
```

Comment:
The card width should be 200px according to the task description. Change it from 198px to 200px.

Reply:
to achieve correct size add `box-sizing: border-box;` to `*` selector